config/initializers/wrap_parameters.rb

```diff
@@ -0,0 +1,14 @@
+# Be sure to restart your server when you modify this file.
+
+# This file contains settings for ActionController::ParamsWrapper which
+# is enabled by default.
+
+# Enable parameter wrapping for JSON. You can disable this by setting :format to an empty array.
```

Comment:
__[rubocop]__
* Metrics/LineLength: Line is too long. [96/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)